src/context.ts

```diff
@@ -131,7 +131,7 @@ export class Context<E extends WebhookPayloadWithRepository = any> implements We
   public issue<T> (object?: T) {
     const payload = this.payload
     return Object.assign({
-      number: (payload.issue || payload.pull_request || payload).number
+      issue_number: (payload.issue || payload.pull_request || payload).number
```

Comment:
to further explain: `context.issue()` would now return `issue_number` instead of `number`. This would not be a problem if passed directly into an Octokit endpoint method, but if people used the returned `number` in other ways, then it would become a problem.

And we cannot return both because that would pass both the new parameter name as well as the deprecated one, and that currently throws an error in Octokit